charts/admission-controller/README.md

```diff
@@ -141,10 +141,12 @@ The following table lists the configurable parameters of the `admission-controll
 | webhook.v2.image.pullPolicy                        | The PullPolicy for KSPM Admission Controller image                                                                                                                                                                                                                                                                                                                                                                                                            | <code></code>                                                                                                                                                                                      |
 | webhook.name                                       | The service name for Webhook deployment                                                                                                                                                                                                                                                                                                                                                                                                                       | <code>webhook</code>                                                                                                                                                                               |
+| webhook.vm.enabled                                 |                                                                                                                                                                                                                                                                                                                                                                                                                                                               | <code>true</code>                                                                                                                                                                                  |
+| webhook.vm.clusterScannerEndpoint                  | NOTE: If you're installing ClusterScanner via ClusterShield the clusterScannerEndpoint will be in the form `${cluster-shield-installation-fullname}-cluster-scanner:9999` Port is always 9999                                                                                                                                                                                                                                                                 | <code>"CHANGEME:9999"</code>                                                                                                                                                                       |
```

Comment:
Is this something that needs to be manually configured by the customer? Can we have a safe default instead of CHANGEME:9999?

Reply:
not really, see https://github.com/sysdiglabs/charts/pull/1651#discussion_r1541502169

Reply:
will add a specific check for this then, as agreed by voice